src/Commands/CommercePrintfulCommands.php

```diff
@@ -89,12 +71,23 @@ public function test() {
     try {
       $store_data = $this->pf->getStoreInfo();
```

Comment:
This will not work with the new config structure (probably you still have the old config with the API key). You need to set the API key (from a store config entity) with `$pf-> setConnectionInfo()`.

Reply:
I need to update the service as well not to get the API key from old config and throw an exception if it's not set.

Reply:
Ok, that's interesting, didn't think about the old api key still being in config.  Will fix and hopefully update tomorrow.  Thanks for the quick review.

Reply:
Right: https://github.com/graber-1/commerce_printful/commit/53156c5b67d486d07220a4ae8b5921f847c5d5ae

Reply:
Made the requested changes, sorry for the merge conflict :)